scripts/test_filtering.py

```diff
@@ -98,8 +98,12 @@
 def filter_integer_tests(input_args):
     multi_bit_filter = "_multi_bit" if input_args.multi_bit else ""
     backend_filter = ""
+    gpu_params_filter = ""
```

Comment:
could we maybe update the multibit filter depending on the gpu input param ?

Reply:
You're right!